helm/chart/router/Chart.yaml

```diff
@@ -19,10 +19,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0-rc.2
+version: 1.0.0-rc.3
```

Comment:
Just want to call out that is very intentionally not bumped while some outstanding work is finished up and stabilized.  It will eventually graduate to 1.0.0 as well!